src/geophires_x/WellBores.py

```diff
@@ -11,6 +11,82 @@
 from .OptionList import ReservoirModel
 
 
+def InjectionReservoirPressurePredictor(project_lifetime: int, timesteps_per_year: int, initial_pressure: float,
+                               inflation_rate: float) -> list:
```

Comment:
These parameters should all be quantities so units are preserved, or at the very least have names that are suffixed with the unit i.e. `project_lifetime_years`, `initial_pressure_MPa`, etc.

Reply:
agreed. changed.